content/docs/concepts/traffic-management/traffic-routing/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Traffic Routing
+description: Describes traffic routing on Istio using service subsets and provides the most common uses.
+weight: 8
+keywords: [routing, request, traffic routing, requests routing, A/B testing, splitting, circuit breakers, subsets, subset]
+aliases:
+    - /docs/concepts/traffic-management/pilot
+---
+
+As described in [Pilot's architecture](../pilot), Pilot maintains an
+`Abstract Model` with the canonical representation of the services in a mesh.
+Istio's model of a service is independent of its representation in the
+underlying platform: Kubernetes, Mesos, Cloud Foundry, etc. The
+platform-specific adapters populate the `Abstract Model` representation with
+various fields from the metadata in the platform.
+
+To cover common configuration scenarios, Istio introduces the concept of a
+**service subset**, to subdivide service instances by versions, say `v1` and
+`v2`, or environment, say `staging` and `prod`. These variants are not
+necessarily different API versions: they can be iterative changes to the same
+service, deployed in different environments like prod, staging, dev, etc.
+Together with Istio's network configuration objects, you can use subsets to
```

Comment:
Once you've defined your service subsets, you can use these subsets in Istio's network configuration objects, for example to direct a particular type of traffic to a particular service version in a [virtual service](), or to specify security policies for a particular service version in a [destination rule]().